index.js

```diff
@@ -4,7 +4,7 @@ function build (schema) {
   /*eslint no-new-func: "off"*/
   var code = `
     'use strict'
-
+    const properties = ${JSON.stringify(schema.properties)}
```

Comment:
Nice one, I completely missed this. Can you please explicit why this is needed with a comment?